iocore/net/SSLConfig.cc

```diff
@@ -66,6 +65,12 @@ init_ssl_ctx_func SSLConfigParams::init_ssl_ctx_cb          = nullptr;
 load_ssl_file_func SSLConfigParams::load_ssl_file_cb        = nullptr;
 IpMap *SSLConfigParams::proxy_protocol_ipmap                = nullptr;
 
+#if TS_HAS_TLS_EARLY_DATA
+#define EARLY_DATA_DEFAULT_SIZE 16384
+int SSLConfigParams::server_max_early_data      = 0;
```

Comment:
I would use the type uint32_t, since that is the type on the OpenSSL calls:
`int SSL_set_max_early_data(SSL *s, uint32_t max_early_data);`

Reply:
changed